src/app/content/components/Page/highlightManager.ts

```diff
@@ -58,7 +57,7 @@ const onSelectHighlight = (
   services.setPendingHighlight(highlight);
 });
 
-const createHighlighter = (services: Omit<Services, 'highlighter'>) => {
+export const createHighlighter = (services: Omit<Services, 'highlighter'>) => {
```

Comment:
why are you exporting this?

Reply:
At first, I was trying to mock this, but then I realized it is not needed and forget to remove this.